superset/assets/src/glossary.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { t } from '@superset-ui/translation';
+import InlineDefinition from './components/InlineDefinition';
+
+const words = {
+  datasource: {
+    word: t('datasource'),
+    definition: t(`A **datasource** is a Superset abstraction that points to
+      specific **table** or **view**, and extends it with **extra metadata** like
+      metrics definition, calculated columns and other relevant **settings
+      and properties**.`),
```

Comment:
Will embedding markdown into translate-able texts play nicely when translated into other languages? I wonder if it will be apparent to translators that they'll need to retain formatting?

Reply:
I think `markdown` can really be part of the solution. One thing is clear: cutting into a bunch of smaller strings isn't the right approach.